gnuplot.el

```diff
@@ -3206,7 +3205,7 @@ maintainer of `gnuplot-mode'."
 This was written with version 4.6 of gnuplot in mind, but should
 work with newer and older versions.
 
-Report bugs in `gnuplot-mode' using \\[gnuplot-bug-report].
+Report bugs in `gnuplot-mode' as issues at https://github.com/emacsorphanage/gnuplot
```

Comment:
How about 

```
Report bugs in `gnuplot-mode' at https://github.com/emacsorphanage/gnuplot/issues
```

or even just

```
Report bugs at https://github.com/emacsorphanage/gnuplot/issues
```

Reply:
Did you see this?

Reply:
Yes.
But I recheck my commit, I've forgotten to apply your suggestions in some places.